keyboards/handwired/rs60/keymaps/default/keymap.c

```diff
@@ -0,0 +1,47 @@
+#include QMK_KEYBOARD_H
+
+enum layers {
+  _QWERTY,
+  _HYPER,
+};
+
+#define KC_ KC_TRNS
+
+#define KC_ESCC MT(MOD_LCTL, KC_ESC)
+#define KC_RST  RESET
+#define KC_BSPH LT(_HYPER, KC_BSPC)
+#define KC_ENTS MT(MOD_LSFT, KC_ENT)
+// Brightness
+#define KC_BRUP KC_PAUS
+#define KC_BRDN KC_SLCK
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  [_QWERTY] = LAYOUT_kc( \
```

Comment:
Both keymap layers should be formatted to use the `LAYOUT` macro from `rs60.h`, rather than `LAYOUT_kc`.

Reply:
We don't want `LAYOUT_kc` in the default keymap.  If you want to add another keymap folder that uses, that's fine.

And it's not friendly.  It's visually appealing. That's it. 

We've had more than a few "newbs" have serious issues with the `KC_KEYMAP` and `LAYOUT_kc`, because you have to add a bunch of defines to make sure everything starts with `KC_`.  

Because of this, it's much easier to not use it for the default keymap, at all.